examples/xgboost_simple.py

```diff
@@ -28,16 +30,35 @@ def objective(trial):
     param = {
         "verbosity": 0,
         "objective": "binary:logistic",
+        #use exact for small dataset
+        "tree_method": "exact",
+        #evaluation metric
+        "eval_metric": trial.suggest_categorical("eval_metric", ["rmse", "rmsle", "logloss","map"]),
```

Comment:
I'm not sure why the evaluation metric is sampled as a hyper-parameter. If the task is classification, I think that `logloss` is okay, but the others are unusual evaluation.

Reply:
I tried directly keeping it <code> logloss </code> and it works fine, but I have also tried different <code> "eval_metric"</code> given on [this page](https://xgboost.readthedocs.io/en/latest/parameter.html), which gives about same results, so I just wanted to make sure that users are aware of available options when trying on xgboost example which can expand the search space.